src/skel/glfw/glfw.cpp

```diff
@@ -233,7 +233,9 @@ psGrabScreen(RwCamera *pCamera)
  *****************************************************************************
  */
 #ifdef _WIN32
+#ifdef _MSC_VER
```

Comment:
This syntax is also used in `src/audio/oal/stream.cpp`.
```suggestion
#if defined _MSC_VER && !defined CMAKE_NO_AUTOLINK
```

Reply:
Grazie grazie.